app/recurring_job.go

```diff
@@ -505,28 +453,28 @@ func (job *Job) getVolumeHeadSize(volume *longhornclient.Volume) (int64, error)
 	return volumeHeadSize, nil
 }
 
-func (job *Job) listSnapshotNamesForCleanup(snapshots []longhornclient.Snapshot, backupDone bool) []string {
+func (job *Job) listSnapshotNamesForCleanup(snapshotCRs []longhornclient.SnapshotCR, backupDone bool) []string {
```

Comment:
TODO: I will review this part after [this PR](https://github.com/longhorn/longhorn-manager/pull/1672) merge